example/lib/src/routing/parser.dart

```diff
@@ -10,13 +10,17 @@ class TemplateRouteParser extends RouteInformationParser<RouteDefinition> {
 
   RouteState? routeState;
 
-  TemplateRouteParser(this.initialRoute, {
+  TemplateRouteParser(
+    this.initialRoute, {
+
     /// The list of allowed routes
     required List<RouteDefinition> routeDefinitions,
-  })  :_routeDefinitions = [
+  })  : _routeDefinitions = [
```

Comment:
The two spaces before the `:` here seems weird, and I'd expect the linter to complain, but I know nothing about dart formatting conventions 🤷‍♂️